pkg/secrets/service/manager.go

```diff
@@ -53,6 +54,15 @@ func (c *serviceCrypter) EncryptValue(ctx context.Context, plaintext string) (st
 	return base64.StdEncoding.EncodeToString(ciphertext), nil
 }
 
+func (*serviceCrypter) SupportsBulkEncryption() bool {
+	// TODO: Check if the service supports bulk encryption.
+	return false
+}
+
+func (c *serviceCrypter) BulkEncrypt(ctx context.Context, secrets []string) ([]string, error) {
+	return nil, errors.New("Not implemented")
+}
```

Comment:
This is awaiting me to make the parallel PR into the service before finalizing this part.